pjsip/include/pjsip/sip_util.h

```diff
@@ -322,7 +322,7 @@ pjsip_endpt_create_request_from_hdr( pjsip_endpoint *endpt,
  * will construct all the Via, Record-Route, Call-ID, From, To, CSeq, and 
  * Call-ID headers from the request.
  *
- * Note: the txdata reference counter is set to ZERO!.
+ * Note: the txdata reference counter is set to unity!
```

Comment:
Minors:
- What if just use the same wordings as `pjsip_endpt_create_request*()`, i.e.: `Once a transmit data is created, the reference counter is initialized to 1.`
- Also while in this, perhaps better to also add the same info for `pjsip_endpt_create_ack()` & `pjsip_endpt_create_cancel()`.